uaclient/cli.py

```diff
@@ -1549,6 +1549,7 @@ def action_status(args, *, cfg):
     if not cfg:
         cfg = config.UAConfig()
     show_beta = args.all if args else False
+    show_all = args.all if args else False
```

Comment:
With this changeset, `show_beta` and `show_all` are coming from the same flag (`--all`) and there is no difference between them. I think we should either:
1. Create a new flag, `--beta`, and let it take care only of beta services, and then `--all` would mean unavailable+beta services, or
2. Remove the `show_beta` variable, as it is only depending on show_all here anyway when running from the CLI. Then, the only way to show beta without showing the unavailable services would be the `allow_beta` feature.

I would be inclined to 2 for simplicity purposes, but this is not a strong opinion.

Reply:
I agree with (2). No need to arg-drill two separate variables equal to the same value.

When we get to the function that handles `allow_beta` from the config, we can give that the value of `show_all` for `show_beta` (I don't remember the exact place where this happens, so that might not totally make sense)

Reply:
It does - `_handle_beta_services` is receving `show_all or show_beta` in the changeset, so yeah it works

Reply:
That makes sense, I will change that